envi/archs/amd64/__init__.py

```diff
@@ -140,6 +140,28 @@ def i_pextrd_q(self, op):
         else:
             self.i_pextrb(op, width=4)
 
+    def i_div(self, op):
```

Comment:
So I think if you change these lines: 
https://github.com/vivisect/vivisect/blob/master/envi/archs/i386/emu.py#L963 (single slash to double to avoid some rounding issues python seems to be doing)
https://github.com/vivisect/vivisect/blob/master/envi/archs/i386/emu.py#L966 (64 -> 128)

in the i386 emulator, we can make deduplicate this condition from that one. You might have to change out the single slashes in the i386 cases as well.

Your thoughts?

Reply:
I was putting it in amd64/__init__.py next to i_idiv, so they're both in the same place.  It would be fewer lines of code to mix it in the i386, but personally I think it's clearer to separate the 64-bit method since it is a little different.

Reply:
I only say to mix it into the i386 since the base i386 emulator still has the buggier case for the 64-bit version. I'm cool with moving it to live in amd64, but I do want to reduce the amount of dead code in the codebase. So if you want it to live in amd64, can you remove the case in i386 so we don't have that case anymore? It keeps things a little tidier.

Reply:
My apologies for the long delay in responding; I was not getting email notifications for one thing.

I removed the dead code in i386 and added another unit test case for dividing by eax.

Reply:
Sweet. lgtm. Thank you!